platform/features/plot/src/PlotController.js

```diff
@@ -254,6 +258,18 @@ define(
                 new PlotAxis("ranges", [], AXIS_DEFAULTS[1])
             ];
 
+            $scope.exportPDF = function () {
+                PlotController.prototype.exportPDF(self.$element, 'plot.pdf');
+            };
+
+            $scope.exportPNG = function () {
+                PlotController.prototype.exportPNG(self.$element, 'plot.png');
+            };
+
+            $scope.exportJPG = function () {
+                PlotController.prototype.exportJPG(self.$element, 'plot.jpg');
+            };
+
```

Comment:
PlotController is exposed as `plot` in the Angular scope already, so I believe these could be omitted by instead having `ng-click="plot.exportPDF()"` in the plot template. This would make the code and changeset simpler.